addons/host-upgrades/resources/40-configmap.yml

```diff
@@ -0,0 +1,159 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  namespace: kube-system
+  name: host-upgrades
+  labels:
+    app: host-upgrades
+data:
+  yum-cron.conf: |
+    [commands]
+    #  What kind of update to use:
+    # default                            = yum upgrade
+    # security                           = yum --security upgrade
+    # security-severity:Critical         = yum --sec-severity=Critical upgrade
+    # minimal                            = yum --bugfix update-minimal
+    # minimal-security                   = yum --security update-minimal
+    # minimal-security-severity:Critical =  --sec-severity=Critical update-minimal
+    update_cmd = default
```

Comment:
I believe this will install all available updated package from all yum repos... my understanding is that CentOS is missing the required repo metadata for things like `minimal-security` to work, those are only available in RHEL:

* https://www.reddit.com/r/CentOS/comments/7wrjcp/how_to_handle_security_upgrades/
* https://updateinfo.cefs.steve-meier.de/
* https://www.caseylabs.com/centos-automatic-security-updates-do-not-work/

Reply:
Yes, CentOS is missing security metadata. We lock kube/docker packages with `yum-lockversion` which should make this a bit more safe. For RHEL we could use `security` or similar?

Reply:
Should probably still consider testing the combination of `yum-lockversion` and the `yum-cron` configuration to make sure it doesn't upgrade pinned packages.